app/views/facilities/edit.html.erb

```diff
@@ -1,25 +1,28 @@
 <%=form_for @facility do |f|%>
```

Comment:
It might be helpful to add labels for all of the input fields in your forms so that the use can still see "City" after they've typed in "Durham".